doc/api/module.md

```diff
@@ -111,6 +111,15 @@ changes:
 Register a module that exports [hooks][] that customize Node.js module
 resolution and loading behavior. See [Customization hooks][].
 
+> **Warning** If setting up a `MessageChannel` to communicate with a loader
+> within the Module Worker, beware that this can lead to a deadlock. For
```

Comment:
```suggestion
> **Warning** If setting up a `MessageChannel` to communicate with hooks, beware that this can lead to a deadlock. For
```

Are you referring to the thread that we create for hooks?

Reply:
Yes. Do we not have an official name for it?